redis.go

```diff
@@ -309,6 +301,13 @@ func NewClient(opt *Options) *Client {
 	return newClient(opt, newConnPool(opt))
 }
 
+func (c *Client) Clone() *Client {
```

Comment:
Can we unexport this?

Please rename `Clone` to `Copy`. I think this is more idiomatic name :)